portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Configuration/RuntimeConfiguration.jsx

```diff
@@ -293,6 +295,9 @@ export default function RuntimeConfiguration() {
             case 'saveButtonDisabled':
                 setSaveButtonDisabled(value);
                 return state;
+            case 'enableSubscriberVerification':
```

Comment:
We don't need to repeat the state update logic here. You can add a case similar to how 'tags' are handled around line 199

Reply:
Fixed with https://github.com/wso2/apim-apps/compare/7ac7ff3e05faf284c2d238fab730ef6add2d4832..0bab035371a7d5842fb8a169b89bfcb1cbf6e6f0